contracts/exchange/SmartWalletSwapImplementation.sol

```diff
@@ -599,6 +661,20 @@ contract SmartWalletSwapImplementation is SmartWalletSwapStorage, ISmartWalletSw
         );
     }
 
+    function checkUserDebt(bool isV1, address token, uint256 amount) internal view returns (uint256) {
+        uint256 debt = lendingImpl.getUserDebt(
```

Comment:
should pass the platform into the lendingImpl function call

Reply:
do we support this feature for compound?

Reply:
Yeah we will also support Compound, so updating this.